app/definitions/rest/v1/subscriptions.ts

```diff
@@ -0,0 +1,5 @@
+export type SubscriptionsEndpoints = {
+	'subscriptions.unread': {
+		POST: (params: { firstUnreadMessage: { _id: string } }) => { success: boolean };
```

Comment:
```suggestion
		POST: (params: { firstUnreadMessage: { _id: string } }) => {};
```
Just remove this success, the rest is fine.